apps/wallet-dashboard/lib/utils/migration/groupMigrationObjects.ts

```diff
@@ -42,9 +43,12 @@ export async function groupMigrationObjectsByUnlockCondition(
             const objectFields = extractOutputFields(object);
 
             let groupKey: string | undefined;
+
             if (groupByTimelockUC) {
-                const timestamp = objectFields.timelock_uc?.fields.unix_time.toString();
-                groupKey = timestamp;
+                const timestamp =
+                    objectFields.timelock_uc &&
+                    objectFields.timelock_uc.fields.unix_time + SECONDS_PER_DAY;
```

Comment:
the logic to display one date or another is more of a UI thing, I wouldnt change the grouping logic